content/en/docs/instrumentation/python/automatic/operator.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Using the OpenTelemetry Operatory to Inject Auto-Instrumentation
+linkTitle: Operator
+weight: 45
+spelling: cSpell:ignore distro mkdir uninstrumented virtualenv
+---
+
+If running your python service in kubernetes you can take advantage of the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator) to inject auto-instrumentation without having to make changes to your service's dependencies to run command.
+
+## Setup
+
+First you must install the [OpenTelemetry Operator](https://github.com/open-telemetry/opentelemetry-operator) into your cluster.
+
+This can be done directly via the [Operator release manifest](https://github.com/open-telemetry/opentelemetry-operator#getting-started), via the [Operator helm chart](https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-operator#opentelemetry-operator-helm-chart), or via [Operator Hub](https://operatorhub.io/operator/opentelemetry-operator).  
+
+In most cases you will need to the [cert-manager installed](https://cert-manager.io/docs/installation/), but if using the helm chart there is an option to generate a self-signed cert.
+
+## Create an OpenTelemetry Collector (Optional)
+
+It is a best practice to send telemetry from containers to an [OpenTelemetry Collector](../../../../collector/) instead of directly to a backend.  The Collector helps simplify secret management, provides reliability and scalability, and can help improve data (k8sattributes processor).  If you chose not to use a Collector you can skip to the next section.
```

Comment:
```suggestion
It is a best practice to send telemetry from containers to an [OpenTelemetry Collector](../../../../collector/) instead of directly to a backend.  The Collector helps simplify secret management, decouples data export problems (such as a need to do retries) from your apps, and lets you add additional data to your telemetry, such as with the `k8sattributesprocessor` component. If you chose not to use a Collector, you can skip to the next section.
```